tests/ISA/ISA.Spreadsheet.Tests/Performance.Tests.fs

```diff
@@ -19,6 +19,24 @@ let private tests_Study = testList "Study" [
         convertToArcFile fswb
 ]
 
+let private tests_Investigation = testList "Investigation" [
+    testCase "WriteManyStudies" <| fun _ ->
+        let inv = ArcInvestigation.init("MyInvestigation")
+        for i = 0 to 1500 do 
+            let s = ArcStudy.init($"Study{i}")
+            inv.AddRegisteredStudy(s)
+        let timer = Stopwatch()
+        timer.Start()
+        let wb = ArcInvestigation.toFsWorkbook inv
+        timer.Stop()
+        let runtime = timer.Elapsed.Milliseconds
+        let expected = 1000 // this is too high and should be reduced
+
+        Expect.equal (wb.GetWorksheets().Count) 1 "Worksheet count"
+        Expect.isTrue (runtime <= expected) $"Expected conversion to be finished in under {expected}, but it took {runtime}"
```

Comment:
can you abstract this as `Expect.isFastenThan: func1: unit -> unit * func2: unit -> unit`, which runs both functions and fails if func2 is faster than func 1?
Would be much appreciated and could be added to Pyxpecto